docs/orchestrating-elastic-stack-applications/elasticsearch/users-and-roles.asciidoc

```diff
@@ -0,0 +1,154 @@
+:parent_page_id: elasticsearch-specification
+:page_id: users-and-roles
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{parent_page_id}.html#k8s-{page_id}[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Users and roles
+
+== Default elastic user
+
+When the Elasticsearch resource is created, a default user named `elastic` is created automatically, and is assigned the `superuser` role.
+
+Its password can be retrieved in a Kubernetes secret, whose name
+is based on the Elasticsearch resource name: `<elasticsearch-name>-es-elastic-user`.
+
+For example, the password of the `elastic` user for an Elasticsearch cluster named `quickstart` can be retrieved with:
+
+[source,sh]
+----
+kubectl get secret quickstart-es-elastic-user -o=jsonpath='{.data.elastic}' | base64 --decode
```

Comment:
Not that it matters much but `kubectl get secret quickstart-es-elastic-user -o go-template='{{.data.elastic | base64decode}}' would work without `base64` installed on the users machine

Reply:
Good point, also changing this in the quickstart.